tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_access.c

```diff
@@ -218,12 +317,22 @@ static void common_configure(uint16_t addr)
 		return;
 	}
 
-	for (int i = 0; i < ARRAY_SIZE(models); i++) {
-		err = bt_mesh_cfg_mod_app_bind(0, addr, addr, 0, models[i],
+	for (int i = 0; i < ARRAY_SIZE(model_ids); i++) {
+		err = bt_mesh_cfg_mod_app_bind(0, addr, addr, 0, model_ids[i],
 					       &status);
 		if (err || status) {
 			FAIL("Model %#4x bind failed (err %d, status %u)",
-					models[i], err, status);
+					model_ids[i], err, status);
+			return;
+		}
+	}
+
+	for (int i = 0; i < ARRAY_SIZE(model_ids); i++) {
```

Comment:
No need in this cycle.

Reply:
Each model instance should be bound to the application key. Why do you think the cycle is not required.